edunext_openedx_extensions/manage_api/views.py

```diff
@@ -126,6 +129,51 @@ def post(self, request):
 
         return JsonResponse({"success": True}, status=201)
 
+    def get(self, request, **kwargs):  # pylint: disable=unused-argument
+        """
+        TODO: add me
+        """
+
+        username = request.GET.get('username')
+        email = request.GET.get('email')
+
+        result, status = self._get_user(username, email)
+
+        return JsonResponse(result, status=status)
+
+    def _get_user(self, username, email):
+        """
+        Helper method to return the username and email of an existing user.
+        """
+        try:
+            existing_user = User.objects.get(username=username)
```

Comment:
Do we currently use this API for the enrollment-api? I believe the answer is no. If this is the case, then my follow up question then would be: Are you planning to change the enroll_api to use only this API, or use the user-api at edxapp for the username based requests, and this for the email based requests?